src/analysis/retail/priest/holy/modules/talents/TopRow/SanctifiedPrayers.tsx

```diff
@@ -0,0 +1,127 @@
+import { formatNumber, formatPercentage } from 'common/format';
+import Abilities from 'analysis/retail/priest/holy/modules/Abilities';
+import AbilityTracker from 'analysis/retail/priest/holy/modules/core/AbilityTracker';
+import SPELLS from 'common/SPELLS';
+import TALENTS from 'common/TALENTS/priest';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { BeginCastEvent, CastEvent, HealEvent } from 'parser/core/Events';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import ItemHealingDone from 'parser/ui/ItemHealingDone';
+
+/*
+  Using Holy Word: Sanctify increases the healing done by Prayer of Healing by 15% for 15 sec.
+*/
+
+const SANCTIFIED_PRAYERS_MULTIPLIER = 1.15;
+
+class SanctifiedPrayers extends Analyzer {
+  static dependencies = {
+    abilityTracker: AbilityTracker,
+    abilities: Abilities,
+  };
+  lastSanctifyCastAt: number = 0;
+  lastCastStartWasBuffed: boolean = false;
+  buffedPohCasts: number = 0;
+  rawAdditionalHealing: number = 0;
+  effectiveAdditionalHealing: number = 0;
+  overhealing: number = 0;
+  protected abilityTracker!: AbilityTracker;
+  protected abilities!: Abilities;
+
+  constructor(options: Options) {
+    super(options);
+    this.active = this.selectedCombatant.hasTalent(TALENTS.SANCTIFIED_PRAYERS_TALENT.id);
+
+    if (this.active) {
+      this.addEventListener(
+        Events.cast.by(SELECTED_PLAYER).spell(TALENTS.HOLY_WORD_SANCTIFY_TALENT),
+        this.sanctifyCast,
+      );
+      this.addEventListener(
+        Events.begincast.by(SELECTED_PLAYER).spell(TALENTS.PRAYER_OF_HEALING_TALENT),
+        this.startPohCast,
+      );
+      this.addEventListener(
+        Events.cast.by(SELECTED_PLAYER).spell(TALENTS.PRAYER_OF_HEALING_TALENT),
+        this.finishPohCast,
+      );
+      this.addEventListener(
+        Events.heal.by(SELECTED_PLAYER).spell(TALENTS.PRAYER_OF_HEALING_TALENT),
+        this.onHeal,
+      );
+    }
+  }
+
+  get unbuffedPohCasts() {
+    return (
+      this.abilityTracker.getAbility(TALENTS.PRAYER_OF_HEALING_TALENT.id).casts -
+      this.buffedPohCasts
+    );
+  }
+
+  get percentOverhealing() {
+    if (this.rawAdditionalHealing === 0) {
+      return 0;
+    }
+    return this.overhealing / this.rawAdditionalHealing;
+  }
+
+  sanctifyCast(event: CastEvent) {
+    this.lastSanctifyCastAt = event.timestamp;
+  }
```

Comment:
i don't see this being used. you could remove this event listener, handler, and the `lastSanctifyCastAt` variable